### ISSUE TITLE
Make overwrite confirmation dialog keyboard-accessible in `Export Program...`

### ISSUE THREAD

**Issue by gemesa** [NationalSecurityAgency/ghidra#8571]
**Is your feature request related to a problem? Please describe.**
When I patch a binary in Ghidra and want to export it via `Export Program...`, I usually do the following:
- press `O` (`Export Program...` shortcut)
- hit enter to press `OK`
- then click on`Overwrite` with my mouse

My problem is that the overwrite popup window is not in focus (doesn't have keyboard focus?) and pressing enter does nothing here, so I have to use the mouse which is frustrating when I have to do this many times.

<img width="246" height="673" alt="Image" src="https://github.com/user-attachments/assets/871b8eba-d716-40ee-a215-59fbef650666" />

<img width="666" height="316" alt="Image" src="https://github.com/user-attachments/assets/5ccd5770-ce36-407f-8f18-e6a5664773b0" />

<img width="718" height="254" alt="Image" src="https://github.com/user-attachments/assets/397dd131-dcd2-43e5-af1a-17a5ffb21df8" />

**Describe the solution you'd like**
Pressing enter should accept that the file will be overwritten.

**Describe alternatives you've considered**
Click on the button with my mouse.

**Additional context**
N/A

**Comment by dragonmacher** [comment]
This seems to be working for me.   The overwrite dialog does have focus for me when it pops up.   If it did not, then that would be an issue.  Is this consistently failing for you?

**Comment by gemesa** [comment]
This is my workflow:
- navigate in the listing view and patch some instructions
- leave my cursor in the listing view
- press `O`
- press `Enter` --> `OK` is pressed
- press `Enter` --> `Overwrite...` is not pressed, so I need to click on it manually

I tested this behavior on Windows and macOS and it does not work on either. On WSL Linux it seems to work fine though.

**Comment by gemesa** [comment]
Awesome, thanks for the fix.
